dependencyManagement/build.gradle.kts

```diff
@@ -75,8 +75,8 @@ val DEPENDENCIES = listOf(
   // using old version of okhttp to avoid pulling in kotlin stdlib
   // not using (old) okhttp bom because that is pulling in old guava version
   // and overriding the guava bom
-  "com.squareup.okhttp3:okhttp:3.12.13",
-  "com.squareup.okhttp3:okhttp-tls:3.12.13",
+  "com.squareup.okhttp3:okhttp:3.14.9",
```

Comment:
We were accidentally not using the latest version (.12.13 appeared newer but was actually a backport of a fix to older okhttp3)